gregorio-menu.js

```diff
@@ -5,53 +5,53 @@ function menuline(str){
 function write_gre_menu() {
 menuline('<div class="menu">');
 menuline('    <ul class="menuitem">');
-menuline('      <li><a href="/">Gregorio</a>');
+menuline('      <li><a href="./">Gregorio</a>');
```

Comment:
If you agree with omitting the `./` for relative links, this can be `href="index.html"`.